keel-web/src/test/resources/examples/ec2-cluster-with-autoscaling-example.yml

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
<3 but you need to add this file explicitly to https://github.com/spinnaker/keel/blob/master/keel-web/src/test/kotlin/com/netflix/spinnaker/keel/rest/ConvertExampleFilesTest.kt

Come to think of it, I bet there's fancy parsing we could do to get things added automatically when they're in this directory... an improvement for another day.

Reply:
Thanks! I didn't even realize these files were consumed by tests, will fix.